.lintr

```diff
@@ -2,5 +2,5 @@ linters: linters_with_defaults(
   line_length_linter = line_length_linter(120),
   cyclocomp_linter = NULL,
   object_usage_linter = NULL,
-  indentation_linter = NULL
+  object_name_linter = object_name_linter(styles = c("snake_case", "symbols"), regexes = c(ANL = "^ANL_?[0-9]*$", ADaM = "^r?AD[A-Z]{2,3}_?[0-9]*$"))
```

Comment:
There are some `# nolint object_name` that can be removed from the code

Reply:
I can whip a quick PR that removes them

![image](https://github.com/insightsengineering/teal.modules.general/assets/211358/c417be01-65fa-460a-820f-3fd99167c862)

Reply:
Maybe add `ADSL` and friends while you're at it?

Reply:
Created here: https://github.com/insightsengineering/teal.modules.general/pull/703

Targets this branch

Reply:
Number of `nolint` hints was drastically reduced

![image](https://github.com/insightsengineering/teal.modules.general/assets/211358/632341ce-6d48-42ab-b710-68bb126d7dca)

Reply:
a true hero that we do not deserve

Reply:
Do you think you could do the same fot `tmc`?